great_expectations/validator/validator.py

```diff
@@ -413,6 +404,28 @@ def get_metrics(self, metrics: Dict[str, MetricConfiguration]) -> Dict[str, Any]
             for (metric_name, metric_configuration) in metrics.items()
         }
 
+    @staticmethod
+    def _get_default_domain_kwargs(metric_provider_cls, metric_configuration):
+        for key in metric_provider_cls.domain_keys:
+            if (
+                key not in metric_configuration.metric_domain_kwargs
+                and key in metric_provider_cls.default_kwarg_values
+            ):
+                metric_configuration.metric_domain_kwargs[
+                    key
+                ] = metric_provider_cls.default_kwarg_values[key]
+
+    @staticmethod
+    def _get_default_value_kwargs(metric_provider_cls, metric_configuration):
```

Comment:
Would you mind type annotating the args and return value? Just to be consistent with the rest of the functions in the file 😄